initiateCO_backend/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+ASGI config for initiateCO_backend project.
+
+It exposes the ASGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'initiateCO_backend.settings')
+
+application = get_asgi_application()
```

Comment:
Since we're not testing anything and decided to use flask instead of django, could we remove this?

Reply:
These files should be gone